subsys/net/lib/ptp/clock.c

```diff
@@ -506,8 +507,21 @@ int ptp_clock_management_msg_process(struct ptp_port *port, struct ptp_msg *msg)
 	return state_decision_required;
 }
 
+static double ptp_servo_pi(int64_t nanosecond_diff)
+{
+	double kp = 0.7;
+	double ki = 0.3;
```

Comment:
Is there any specific reason to use 0.7 and 0.3 value? How this parameters has been tuned?

Reply:
I refered to early linuxptp project. (see below.)

```
#define HWTS_KP 0.7
#define HWTS_KI 0.3
```

The parameters suit for hw timestamping system. If users want to get the best synchronization offset curve, they can tune to get best parameters by themself.
Thanks.